### ISSUE TITLE
error when loading CSs if OPTIONAL variable is not found

### ISSUE THREAD

**Issue by mathib** [mathib/fog-demo-app#2]
loading CSs from RDF files, where the variables in the optional pattern does not occur *at least once* results in an error from Comunica:
```
PREFIX gom: <https://w3id.org/gom#>

SELECT DISTINCT ?cs ?to ?matrix WHERE {
	?cs a gom:CartesianCoordinateSystem .
	OPTIONAL {
		?transformation gom:fromCartesianCoordinateSystem ?cs ;
			gom:toCartesianCoordinateSystem ?to ;
			gom:hasTransformationMatrix ?matrix .
	}
}
```
`comunica-browser.js:136386 Uncaught (in promise) Error: Variables '?to,?matrix' are used in the projection result, but are not assigned.`

**Comment by mathib** [comment]
solved by commit https://github.com/mathib/fog-demo-app/commit/36a1194df8d7c71641fe4f3f018a9ab72abbe02f

**Comment by mathib** [comment]
also solved in Comunica 1.13.1
